erts/emulator/beam/utils.c

```diff
@@ -5241,27 +5338,35 @@ static void erts_qsort_helper(byte *base,
                               size_t nr_of_items,
                               size_t item_size,
                               erts_void_ptr_cmp_t compare,
-                              int extra_seed)
+                              Uint32 extra_seed)
 {
-    byte *second_part_base;
-    size_t nr_of_items_second_part;
-
-    if (nr_of_items <= 1) {
-        return;
-    }
-    if (nr_of_items == 2) {
-        if (compare(base, base+item_size) > 0) {
-            erts_qsort_swap(item_size, base, base+item_size);
-        }
+    erts_qsort_partion_array_result partion_info;
+    size_t nr_of_items_in_first_partion;
+    size_t nr_of_items_second_partion;
+    const size_t stdlib_qsort_cut_off = 16;
+    if (nr_of_items <= stdlib_qsort_cut_off) {
```

Comment:
Borde kanske byta namn från `stdlib_qsort_cut_off`.

Reply:
Fixed in commit below